test.js

```diff
@@ -5,4 +5,51 @@ test('main', t => {
 	t.is(cryptoRandomString(0).length, 0);
 	t.is(cryptoRandomString(10).length, 10);
 	t.is(cryptoRandomString(100).length, 100);
+	t.regex(cryptoRandomString(100), /^[a-f\d]*$/); // Sanity check, probabilistic
+});
+
+test('hex', t => {
+	t.is(cryptoRandomString(0, {type: 'hex'}).length, 0);
+	t.is(cryptoRandomString(10, {type: 'hex'}).length, 10);
+	t.is(cryptoRandomString(100, {type: 'hex'}).length, 100);
+	t.regex(cryptoRandomString(100, {type: 'hex'}), /^[a-f\d]*$/); // Sanity check, probabilistic
+});
+
+test('base64', t => {
+	t.is(cryptoRandomString(0, {type: 'base64'}).length, 0);
+	t.is(cryptoRandomString(10, {type: 'base64'}).length, 10);
+	t.is(cryptoRandomString(100, {type: 'base64'}).length, 100);
+	t.regex(cryptoRandomString(100, {type: 'base64'}), /^[a-zA-Z\d/+]*$/); // Sanity check, probabilistic
+});
+
+test('url-safe', t => {
+	t.is(cryptoRandomString(0, {type: 'url-safe'}).length, 0);
+	t.is(cryptoRandomString(10, {type: 'url-safe'}).length, 10);
+	t.is(cryptoRandomString(100, {type: 'url-safe'}).length, 100);
+	t.regex(cryptoRandomString(100, {type: 'url-safe'}), /^[a-zA-Z\d._~-]*$/); // Sanity check, probabilistic
+});
+
+test('characters', t => {
+	t.is(cryptoRandomString(0, {characters: '1234'}).length, 0);
+	t.is(cryptoRandomString(10, {characters: '1234'}).length, 10);
+	t.is(cryptoRandomString(100, {characters: '1234'}).length, 100);
+	t.regex(cryptoRandomString(100, {characters: '1234'}), /^[1-4]*$/); // Sanity check, probabilistic
+});
+
+test('argument errors', t => {
+	t.throws(() => {
+		cryptoRandomString(Infinity);
```

Comment:
Which one should we use though, with brackets and separate line, or without? In `execa` I was suggested the opposite changes and they were accepted in the end.

Reply:
For the record I myself is in favor of brackets+line for better intent indication and readability.

Reply:
> Which one should we use though, with brackets and separate line, or without?

This format. Like you said, it makes the intention clearer. I do this for all code. I only use inline arrow functions when they are meant to return something.

> In execa I was suggested the opposite changes and they were accepted in the end.

That must have been an oversight. Fixed: https://github.com/sindresorhus/execa/commit/f6db2642ada4dc8b1ca6484aad5d0fa105632b9e